ls8/cpu.py

```diff
@@ -58,18 +59,40 @@ def trace(self):
         ), end='')
 
         for i in range(8):
-            print(" %02X" % self.reg[i], end='')
+            print(" %02X" % self.register[i], end='')
 
         print()
 
     def run(self):
-        """Run the CPU."""
-        pass
-
+        #ir = self.ram[self.pc]
+        operand_a = self.ram_read(self.pc+1)
+        print(f'{operand_a} foo')
+        operand_b = self.ram_read(self.pc+2)
+        print(f'{operand_b} bar')
+        self.trace()
+        #print(self.ram[self.pc+1])
+        #print(self.ram[self.pc+2])
+        while self.running:
+            command = self.ram[self.pc]
+            #print(f'{command} this is the command')
+            #LDI - set value of register to integer
+            if command == 130:
```

Comment:
Your code will be more readable if you pull the command codes into a variable. For instance, set `LDI = 0b...` so that you're checking `command == LDI`. Best to set the variable to the binary value for reference later since there is information you can get from looking at how the bits in the instruction code are set.